db/queries.py

```diff
@@ -3,44 +3,62 @@
 
 
 def get_messages_that_contain_word(word: str) -> list[Message]:
-    pass
+    return list(Message.objects.filter(text__icontains=word))
 
 
 def get_untitled_chats() -> list[Chat]:
-    pass
+    return Chat.objects.filter(title__startswith="Untitled")
 
 
 def get_users_who_sent_messages_in_2015() -> list[str]:
-    pass
+    return list(Message.objects.filter(
+        sent__contains=2015).select_related("user").values_list(
+        "user__first_name", "user__last_name"))
 
 
 def get_actual_chats() -> list[Chat]:
-    pass
+    return list(Chat.objects.filter(
+        Q(message__sent__contains=2021)
+        | Q(message__sent__contains=2022)))
 
 
 def get_messages_contain_authors_first_name() -> list[Message]:
-    pass
+    return Message.objects.filter(text__icontains=F("user__first_name"))
 
 
 def get_users_who_sent_messages_starts_with_m_or_a() -> list[User]:
-    pass
+    return User.objects.filter(Q(message__text__istartswith="a")
+                               | Q(message__text__istartswith="m"))
 
 
 def get_delivered_or_admin_messages() -> list[Message]:
-    pass
+    return Message.objects.filter(Q(is_delivered=True)
+                                  | Q(user__username__istartswith="admin"))
 
 
 def get_count_messages_sent_by_first_name(first_name: str) -> int:
-    pass
+    result = Message.objects.filter(
+        user__first_name=first_name).aggregate(Count("id"))
+    return result["id__count"]
 
 
 def get_top_users_by_number_of_the_messages() -> list[User]:
-    pass
+    return User.objects.annotate(
```

Comment:
Тут і в деяких інших місцях, трохи не зрозуміло, яка душка куди відноситься
```suggestion
    return (
       User.objects
       .annotate(num_messages=Count("message"))
       .order_by("-num_messages")[:3]
    )
```
Коли дуже багато чейняться методів краще всього вот так їх оформити, щоб було зрозуміло, що куди